src/en-GB/news/blog/2017/more-recommendations-for-ceph-and-openstack/index.md

```diff
@@ -0,0 +1,132 @@
+
+Obviously, since flavors can be public, but can also be limited to specific tenants, you can set relatively low instance resource quotas in public flavors, and then make flavors with higher resource quotas available to select tenants only.
+
+## Do differentiate Cinder volume types by disk I/O limits
+
+In addition to setting I/O limits on flavors for VMs, you can also influence the I/O characteristics of volumes. You do so by specifying distinct Cinder volume _types_. Volume types are frequently used to enable users to select a specific Cinder backend — say, to stick volumes either on a NetApp box or on RBD, but it’s perfectly OK if you define multiple volume types using the same backend. You can then set characteristics like maximum IOPS or maximum throughput via Cinder QoS specifications. A QoS specification akin to the Nova flavor decribed above — limiting throughput to 100 MB/s and 1,500 IOPS would be created like this:
+
+openstack volume qos create 
+  --consumer front-end
+  --property total\_bytes\_sec\=$((100<<20)) 
+  --property total\_iops\_sec\=1500 
+  "100MB/s-1500iops"
+
+You would then create a corresponding volume type, and associate the QoS spec with it:
+
+openstack volume type create 
+  --public 
+  "100MB/s-1500iops"
+openstack volume qos associate 
+  "100MB/s-1500iops" 
+  "100MB/s-1500iops"
+
+Again, as with Nova flavors, you can make volume types public, but you can also limit them to specific tenants.
+
+## Don't forget about suspend files
+
+When you **suspend** a Nova/libvirt/KVM instance, what really happens is what libvirt calls a **managed save**: the instance's entire memory is written to a file, and then KVM process shuts down. This is actually quite neat because it means that the VM does not consume any CPU cycles nor memory until it restarts, and it will continue right where it left off, even if the compute node is rebooted in the interim.
+
+You should understand that these savefiles are not compressed in any way: if your instance has 16GB of RAM, that's a 16GB file that instance suspension drops into `/var/lib/nova/save`. This can add up pretty quickly: if a single compute node hosts something like 10 suspended instances, their combined save file size can easily exceed 100 GB. Obviously, this can put you in a really bad spot if this fills up your `/var` (or worse, `/`) filesystem.
+
+Of course, if you already have a Ceph cluster, you can put it to good use here too: just deep-mount a CephFS file system into that spot. Here's an Ansible playbook snippet that you may use as inspiration:
+
+\---
+\- hosts:
+  \- compute-nodes
+
+  vars:
+    ceph\_mons:
+      \- ceph-mon01
+      \- ceph-mon02
+      \- ceph-mon03
+    cephfs\_client: cephfs
+    cephfs\_secret: "{{ vaulted\_cephfs\_secret }}"
```

Comment:
Double curly brackets are causing issues as this syntax is used with nunjucks. In this instance it's causing the content between the brackets not to appear.

Can they be escaped?

E.g.
`\{\{ content here \}\}`

This file contains multiple instances of double curly brackets.